src/main/kotlin/de/nycode/bankobot/commands/general/SearchCommand.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2021 BankoBot Contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ */
+
+package de.nycode.bankobot.commands.general
+
+import de.nycode.bankobot.command.command
+import de.nycode.bankobot.command.description
+import de.nycode.bankobot.commands.GeneralModule
+import de.nycode.bankobot.utils.Embeds
+import de.nycode.bankobot.utils.Embeds.editEmbed
+import de.nycode.bankobot.utils.GoogleUtil
+import de.nycode.bankobot.utils.doExpensiveTask
+import de.nycode.bankobot.utils.paginate
+import dev.kord.x.commands.annotation.AutoWired
+import dev.kord.x.commands.annotation.ModuleName
+import dev.kord.x.commands.argument.extension.named
+import dev.kord.x.commands.argument.text.ListArgument
+import dev.kord.x.commands.kord.model.context.KordCommandEvent
+import dev.kord.x.commands.model.command.invoke
+
+@AutoWired
+@ModuleName(GeneralModule)
+fun searchCommand() = command("google") {
+    alias("find", "search", "duckduckgo")
+    description("Sucht dir deinen Scheiß aus dem Internet zusammen.")
+
+    invoke(ListArgument.named("query")) { query ->
+        search(query.joinToString { s -> "$s " })
```

Comment:
change this to query.joinToString(" ")

Reply:
Actually no change the `ListArgument` to a `StringArgument`  then query is just the whole arguments in a string so no join is needed